sites/breadcrumbs/_breadcrumb.scss

```diff
@@ -3,15 +3,20 @@
  for: GCWeb v5
  */
 
-.gcweb-menu ~ #wb-bc {
```

Comment:
There was probably are reason I used a sibling selector. Probably it was for transitioning. Let document that in the breadcrumb documentation and in the release note. Something like: GCWeb vX.0 are not compatible anymore with the pre 2019 megamenu pattern (GCWEB v4.x) that was taking the full page width.

This kind of create a breaking change.